src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator machine started its work.");

```

Comment:
The message in the 'doWork' method should be more informative and indicate that the 'Excavator' is working. For example, 'Excavator is starting its work.'